### ISSUE TITLE
Add Cyphertite to File Storage & Sync

### ISSUE THREAD

**Issue by andr3jx** [nylira/prism-break#1049]
I want to suggest to add [Cyphertite](https://www.cyphertite.com/) to File Storage & Sync. In comparisson to services like SpiderOak, their clients are [full opensource](https://www.cyphertite.com/download.php?os=source). 
Cyphertite is already 2 years old so I wonder why nobody mentioned it here?

**Comment by jscinoz** [comment]
Seems that cyphertite's client is also available in Gentoo's main portage tree:
```
* app-backup/cyphertite
     Available versions:  (~)2.0.2-r5 (~)2.0.4
     Homepage:            https://www.cyphertite.com/
     Description:         High Security, Zero-Knowledge Online Backup
```

While its not ideal that the server side of things is proprietary, everything that runs locally is FOSS, and nothing leaves your machine without being encrypted locally first.

**Comment by yegortimoshenko** [comment]
Source code (last updated in 2014): https://github.com/conformal/cyphertite

News article, "Cyphertite Closes Doors, Alternatives Available", October 2015: https://www.cloudwards.net/news/cyphertite-closes-doors-alternatives-available-8312/
